packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -80,6 +83,135 @@ export 'dart:io'
         WebSocketException,
         WebSocketTransformer;
 
+/// An [IOOverrides] that can delegate to [FileSystem] implementation if provided.
```

Comment:
It looks like this is only used for testing. Should it go somewhere under `test/src`?

Reply:
Moved to test/src/io.dart